interactive_engine/bin/giectl.sh

```diff
@@ -0,0 +1,396 @@
+#!/usr/bin/env bash
+#
+# interactive_engine command tool
+
+set -x
+set -e
+set -o pipefail
+readonly WORKSPACE=${GRAPHSCOPE_HOME}
+
+err() {
+  echo -e "${RED}[$(date +'%Y-%m-%dT%H:%M:%S%z')]: [ERROR]${NC} $*" >&2
+}
+
+warning() {
+  echo -e "${YELLOW}[$(date +'%Y-%m-%dT%H:%M:%S%z')]: [WARNING]${NC} $*" >&1
+}
+
+log() {
+  echo -e "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&1
+}
+
+
+##########################
+# Output useage information.
+# Globals:
+#   None
+# Arguments:
+#   None
+##########################
+usage() {
+cat <<END
+  A script to install dependencies of GraphScope or deploy GraphScope locally.
+
+  Usage: deploy_local [options] [command]
+
+  Options:
+
+    --prefix <path>      install prefix of GraphScope, default is /usr/local
+    --verbose            ouput the debug logging information
+    -h, --help           output help information
+
+  Commands:
+
+    install_deps         install dependencies of GraphScope
+    deploy               deploy GraphScope locally
+END
+}
+
+
+
+set_common_envs() {
+  object_id=$1
+  if [ ! -n "$GRAPHSCOPE_RUNTIME" ]; then
+    export GRAPHSCOPE_RUNTIME=/tmp/graphscope/runtime
+  fi
+
+  if [ -n "$object_id" ]; then
+    export LOG_DIR=$GRAPHSCOPE_RUNTIME/logs/$object_id
+    export CONFIG_DIR=$GRAPHSCOPE_RUNTIME/config/$object_id
+    export PID_DIR=$GRAPHSCOPE_RUNTIME/pid/$object_id
+  fi
+}
+
+# parallel run for k8s
```

Comment:
for these functions, it would be better if you can give detailed desc, and desc for args.

Reply:
these functions are used only one time in script , remove the functions.